honeybee_schema/cli.py

```diff
@@ -0,0 +1,98 @@
+"""Command Line Interface (CLI) entry point for honeybee schema."""
+
+try:
+    import click
+except ImportError:
+    raise ImportError(
+        'click module is not installed. Try `pip install honeybee-core[cli]` command.'
+    )
+
+import sys
+import logging
+import json
+import os
+import importlib
+
+from honeybee_schema.update._base import Updater
+
+
+@click.group()
+@click.version_option()
+def main():
+    pass
+
+
+_logger = logging.getLogger(__name__)
+
+
+@main.command('update-model')
+@click.argument('model-json', type=click.Path(
+    exists=True, file_okay=True, dir_okay=False, resolve_path=True))
+@click.option('--version', '-v', help='Text to indicate the version to which the model '
+              'JSON will be updated (eg. 1.41.2). Versions must always consist of '
+              'three integers separated b periods. If None, the Model JSON will '
+              'be updated to the last release that included a breaking change.',
+              type=str, default=None, show_default=True)
```

Comment:
If there is no default then we probably don't need show_default=True.

```suggestion
              type=str)
```